pkg/util/externalnode/externalnode.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package externalnode
+
+import "antrea.io/antrea/pkg/apis/crd/v1alpha1"
+
+func GenExternalEntityName(externalNode *v1alpha1.ExternalNode) string {
```

Comment:
Will it be used by other pkgs besides ExternalNodeController? If not I prefer to keep into the controller/externalnode or even controller.go for now.

Reply:
Ok, I saw you said it will be used by agent too. Then I am fine with the current way.